staging/src/k8s.io/component-base/featuregate/testing/feature_gate.go

```diff
@@ -46,52 +49,114 @@ func init() {
 // featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.<FeatureName>, true)
 func SetFeatureGateDuringTest(tb TB, gate featuregate.FeatureGate, f featuregate.Feature, value bool) {
 	tb.Helper()
-	detectParallelOverrideCleanup := detectParallelOverride(tb, f)
```

Comment:
did we lose detectParallelOverrideCleanup in this change? we don't want to lose detection of parallel tests interleaving changing the same gates in opposite ways